src/pages/iou/request/step/IOURequestStepScan/index.native.tsx

```diff
@@ -193,41 +172,43 @@ function IOURequestStepScan({
         }
 
         // If the transaction was created from the global create, the person needs to select participants, so take them there.
-        if (isFromGlobalCreate && iouType !== CONST.IOU.TYPE.TRACK_EXPENSE) {
+        if (transaction?.isFromGlobalCreate && iouType !== CONST.IOU.TYPE.TRACK_EXPENSE) {
             Navigation.navigate(ROUTES.MONEY_REQUEST_STEP_PARTICIPANTS.getRoute(iouType, transactionID, reportID));
             return;
         }
 
         // If the transaction was created from the + menu from the composer inside of a chat, the participants can automatically
         // be added to the transaction (taken from the chat report participants) and then the person is taken to the confirmation step.
+
         IOU.setMoneyRequestParticipantsFromReport(transactionID, report);
+
         Navigation.navigate(ROUTES.MONEY_REQUEST_STEP_CONFIRMATION.getRoute(CONST.IOU.ACTION.CREATE, iouType, transactionID, reportID));
-    }, [iouType, report, reportID, transactionID, isFromGlobalCreate, backTo]);
+    }, [iouType, report, reportID, transactionID, transaction?.isFromGlobalCreate, backTo]);
 
     const updateScanAndNavigate = useCallback(
-        (file, source) => {
+        (file: FileObject, source: string) => {
             Navigation.dismissModal();
-            IOU.replaceReceipt(transactionID, file, source);
+            IOU.replaceReceipt(transactionID, file as File, source);
         },
         [transactionID],
     );
 
     /**
      * Sets the Receipt objects and navigates the user to the next page
-     * @param {Object} file
      */
-    const setReceiptAndNavigate = (file) => {
+    const setReceiptAndNavigate = (file: FileObject) => {
         if (!validateReceipt(file)) {
             return;
         }
 
         // Store the receipt on the transaction object in Onyx
         // On Android devices, fetching blob for a file with name containing spaces fails to retrieve the type of file.
         // So, let us also save the file type in receipt for later use during blob fetch
-        IOU.setMoneyRequestReceipt(transactionID, file.uri, file.name, action !== CONST.IOU.ACTION.EDIT, file.type);
+        // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+        IOU.setMoneyRequestReceipt(transactionID, file?.uri ?? '', file.name || '', action !== CONST.IOU.ACTION.EDIT, file.type);
```

Comment:
Why not use null coalesce here?

Reply:
Ok thanks! I wonder if `||` was added for any specific case where we could not use `??`